Emitter/Emitter.cs

```diff
@@ -213,9 +212,9 @@ private void OnDisconnect(object sender, EventArgs e)
         /// <summary>
         /// Connects the emitter.io service.
         /// </summary>
-        public void Connect()
+        public void Connect(string username = null, string password = null)
```

Comment:
I don't think we need a password, since the broker doesn't support it anyway. Username is indeed useful

Reply:
good point, i havent seen it being returned anyway, ill amend it